examples/curve_loss_of_training.py

```diff
@@ -0,0 +1,97 @@
+'''
+Parse the log files of MatchZoo to draw the learning curves of 
+deep learning model training process to observe the training loss
+and metrics on train/dev/test data. With these curves, we can do 
+a better job in model debugging
+@author: Liu Yang (yangliuyx@gmail.com / lyang@cs.umass.edu)
+@author: Thiziri Belkacem (belkacemthiziri@gmail.com)
+'''
+import os
+import sys
+import matplotlib.pyplot as plt
+from matplotlib import rcParams
+
+def draw_train_learning_curve(log_file, log_label):
+    #command = 'curl ' + log_file 
+    #r = os.popen(command)
+    r = open(log_file, 'r')
```

Comment:
Did you close `r` at somewhere? I didn't see it. Why not `with open(file, 'r') as ... `